src/components/dialog/__tests__/dialog.atoms.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { render } from '@testing-library/react'
+import { Dialog } from '../dialog.atoms'
+
```

Comment:
>  How would you answer the two questions above for these "atoms"?

I would say the tests are useless in here because the "atoms" is actually just a proxy component from styled linaria, and we don't have more variant that need to be stored as an atom

**thought:** talking about testing the styled linaria, I think its not achievable for now, since we're mocking the core of linaria, perhaps the only worth to have is something like a visual testing in storybook level

**question:** what about using the styled linaria directly? and nuking the current "proxy atoms"? it will reduce the redundant and not leading a confusion for react user, so they could use the prefixed style `El` directly if they want

```tsx
<ElDialog>
   <ElDialog.Header>test</ElDialog.Header>
    ...
<ElDialog.Container>
```

instead

```tsx
<Dialog>
  <Dialog.Header>header</Dialog.Header>
  ...
<Dialog>
```